packages/desktop-client/src/components/sidebar/Accounts.tsx

```diff
@@ -77,7 +77,7 @@ function Accounts({
     setIsDragging(drag.state === 'start');
   }
 
-  let makeDropPadding = (i, length) => {
+  let makeDropPadding = (i, _length) => {
```

Comment:
```suggestion
  let makeDropPadding = (i) => {
```

Reply:
There are some cases like this one. The function definition includes the `length` parameter which isn't used in the function so my first thought was to remove it althogether.  But we are calling the function in some places an passing that argument like line [145](https://github.com/actualbudget/actual/blob/28e129c7faf79a597d20c61bfe4f1e37f48e62b6/packages/desktop-client/src/components/sidebar/Accounts.tsx#L145) for example.

Which one do you preffer? Removing the parameter from the function definition and fix the call? or leave the parameter prefixed with an `_` so it gets ignored by the rule?

Reply:
If it's not used - we can safely remove it. :)